RELEASE_NOTES.md

```diff
@@ -6,6 +6,27 @@ Chef 15 release notes will be added here as development progresses.
 
 ## Breaking Changes
 
+### Package provider allow_downgrade is now true by default
+
+A package provider without any `allow_downgrade` flag now will allow downgrades, which is opposite from previous versions. This behavior change will mostly affect users of the rpm and zypper package providers.
```

Comment:
@lamont-granquist Is the assertion that this behavior is different from previous versions correct? (This can always be changed to be more accurate.)

Reply:
Yep this is a potentially breaking change for 15.0